packages/gatsby-source-kontent/src/pluginOptionsSchema.ts

```diff
@@ -0,0 +1,39 @@
+
+const pluginOptionsSchema = ({ Joi }: { Joi: any }) => {

```

Comment:
Consider adding return type. Ain't it possible to add any other type than any for Joi?

Reply:
I tried, but it is a bit problematic:
* Import needs to be done from specific index.d.ts file instead of from module export
* Not sure what type is an argument that [Gatsby is sending](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/#pluginOptionsSchema) - it might ba `JoiObject, `AnySchema` or some descendant of it.

For suture reference:
```js
import { JoiObject } from "joi/index"; // First problem

const pluginOptionsSchema = ({ Joi }: { Joi: any /* Second problem */  }): JoiObject => {
```